Src/library/memory_analyzer/widgets/ma_window_imp.e

```diff
@@ -1026,11 +1030,21 @@ feature {NONE} -- Implementation
 		deferred
 		end
 
+	dpi_resize_histogram (a_dpi, a_x, a_y, a_width, a_height: INTEGER)
+			-- Called by `dpi_changed_actions' of `eiffel_histogram'.
+		deferred
+		end
+
 	eiffel_view_frame_size_change (a_x, a_y, a_width, a_height: INTEGER)
 			-- Called by `resize_actions' of `eiffel_view_frame'.
 		deferred
 		end
 
+	eiffel_view_frame_dpi_size_change (a_dpi, a_x, a_y, a_width, a_height: INTEGER)
+			-- Called by `dpi_changed_actions' of `eiffel_view_frame'.
+		deferred
```

Comment:
Maybe here, no deferred, but effective feature calling ` eiffel_view_frame_size_change `

Reply:
yes, then the previous one is not needed for now.